config/software/stunnel.rb

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# TODO: promote to omnibus-software
+name "stunnel"
+default_version "5.38"
+
+license "GPL-2.0"
+license_file "COPYING"
+
+dependency "openssl"
+
+source url:
+"https://www.stunnel.org/downloads/stunnel-#{version}.tar.gz"
+relative_path "stunnel-#{version}"
+
+version "5.38" do
+  source sha1: "2a34256707ce5f5625018cc9a3b2ef4a5f48089a"
```

Comment:
Ah ok, I can update the cache. I saw sha1 in some examples, but 256 makes more sense.